shopyo/modules/page/info.json

```diff
@@ -4,5 +4,5 @@
         "type": "show",
         "fa-icon": "fa fa-store",
         "url_prefix": "/page",
-        "panel_redirect": "/dashboard"
+        "panel_redirect": ""
 }
```

Comment:
The /dashboard is good. As if it is defined, control panel will redirect to dashboard instead of /contact